Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM golang:1.10
```

Comment:
Is it worthwhile to do a build step in this file so you don't have sources in your final image?

Reply:
Ooo good thinking!! 


I probably won't in the interest of time but ta that'd be ideal